onnx_tf/opset_version.py

```diff
@@ -1,249 +1,249 @@
 backend_opset_version = {
-    'a_ten': [],
-    'abs': [1, 2, 3, 4, 5],
-    'add': [1],
-    'affine': [],
-    'and': [1],
-    'arg_max': [1],
-    'arg_min': [1],
-    'average_pool': [1],
-    'batch_normalization': [1],
-    'cast': [1, 2, 3, 4, 5],
-    'ceil': [1, 2, 3, 4, 5],
-    'clip': [1],
-    'concat': [1, 4],
-    'constant': [1],
-    'constant_fill': [],
-    'conv': [1],
-    'conv_transpose': [1],
-    'crop': [],
-    'depth_to_space': [1],
-    'div': [1],
-    'dropout': [1],
-    'elu': [1],
-    'equal': [1],
-    'exp': [1, 2, 3, 4, 5],
-    'f_c': [],
-    'flatten': [1],
-    'floor': [1, 2, 3, 4, 5],
-    'g_r_u': [],
-    'g_r_u_unit': [],
-    'gather': [1, 2, 3, 4, 5],
-    'gemm': [1],
-    'given_tensor_fill': [],
-    'global_average_pool': [1],
-    'global_lp_pool': [1, 2],
-    'global_max_pool': [1],
-    'greater': [1],
-    'hard_sigmoid': [1],
-    'hardmax': [1],
-    'identity': [1, 2, 3, 4, 5],
-    'if': [],
-    'image_scaler': [],
-    'instance_normalization': [],
-    'l_r_n': [1],
-    'l_s_t_m': [1],
-    'leaky_relu': [1],
-    'less': [1],
-    'log': [1, 2, 3, 4, 5],
-    'log_softmax': [1],
-    'loop': [],
-    'loop_index_tensor': [],
-    'lp_normalization': [1],
-    'lp_pool': [],
-    'mat_mul': [1],
-    'max': [1],
-    'max_pool': [1],
-    'max_roi_pool': [],
-    'mean': [1],
-    'mean_variance_normalization': [],
-    'min': [1],
-    'mul': [1],
-    'neg': [1, 2, 3, 4, 5],
-    'not': [1, 2, 3, 4, 5],
-    'or': [1],
-    'p_relu': [1],
-    'pad': [1, 2],
-    'parametric_softplus': [],
-    'pow': [1],
-    'r_n_n': [],
-    'random_normal': [1, 2, 3, 4, 5],
-    'random_normal_like': [1],
-    'random_uniform': [1, 2, 3, 4, 5],
-    'random_uniform_like': [1],
-    'reciprocal': [1, 2, 3, 4, 5],
-    'reduce_l1': [1],
-    'reduce_l2': [],
-    'reduce_log_sum': [],
-    'reduce_log_sum_exp': [1, 2, 3, 4, 5],
-    'reduce_max': [1, 2, 3, 4, 5],
-    'reduce_mean': [1, 2, 3, 4, 5],
-    'reduce_min': [1, 2, 3, 4, 5],
-    'reduce_prod': [1, 2, 3, 4, 5],
-    'reduce_sum': [1, 2, 3, 4, 5],
-    'reduce_sum_square': [1],
-    'relu': [1, 2, 3, 4, 5],
-    'reshape': [1, 5],
-    'scale': [],
-    'scaled_tanh': [],
-    'selu': [1],
-    'shape': [1, 2, 3, 4, 5],
-    'sigmoid': [1, 2, 3, 4, 5],
-    'size': [1, 2, 3, 4, 5],
-    'slice': [1],
-    'softmax': [1],
-    'softplus': [1, 2, 3, 4, 5],
-    'softsign': [1, 2, 3, 4, 5],
-    'space_to_depth': [1],
-    'split': [1, 2],
-    'sqrt': [1, 2, 3, 4, 5],
-    'squeeze': [1, 2, 3, 4, 5],
-    'sub': [1],
-    'sum': [1],
-    'tanh': [1, 2, 3, 4, 5],
-    'thresholded_relu': [1],
-    'tile': [1],
-    'top_k': [1],
-    'transpose': [1, 2, 3, 4, 5],
-    'unsqueeze': [1],
-    'upsample': [],
-    'xor': [1]}
+    u'a_ten': [],
+    u'abs': [1],
```

Comment:
I see you always output unicode string.
Is it affected by certain local environment setting?

Reply:
good catch, i casted the keys of dictionary to strings and now it produces string.